src/test/java/hudson/plugins/jira/selector/ExplicitIssueSelectorTest.java

```diff
@@ -22,7 +22,7 @@
     public void returnsExplicitCollections() throws IOException {
         JiraSession session = mock(JiraSession.class);
         JiraSite site = mock(JiraSite.class);
-        when(site.getSession()).thenReturn(session);
+        when(site.getSession(null)).thenReturn(session);
```

Comment:
Tests also need to be context-aware unfortunately, otherwise they end up testing against globally defined sites.

Reply:
in the context of this particular test it's very different.... we do not even need any mock.. I will clean that